qiskit_machine_learning/algorithms/regressors/qsvr.py

```diff
@@ -0,0 +1,63 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Quantum Support Vector Regressor"""
+import logging
+
+from typing import Optional
+
+from sklearn.svm import SVR
+
+from ...kernels.quantum_kernel import QuantumKernel
+
+logger = logging.getLogger(__name__)
+
+
+class QSVR(SVR):
+    r"""Quantum Support Vector Regressor.
+
+    This class shows how to use a quantum kernel for classification. The class extends
```

Comment:
Would we want to make this text a bit more different that the classifier text (at present its just the final link below to sklearn user guide whose difference would not be clear in the docs either without hovering over the text or clicking on it . Might it be better here to start out `This class shows how to use a quantum kernel for regression` i.e. change to classification to regression at least.